releasetasks/templates/firefox/partner_repacks.yml.tmpl

```diff
@@ -1,11 +1,10 @@
 {% for platform in partner_repacks_platforms %}
 {% set buildername = "release-{}-{}-{}_partner_repacks".format(branch, product, platform) %}
-{% do push_to_releases_extra_upstream_builders.append(buildername) %}
 -
     taskId: "{{ stableSlugId(buildername) }}"
     reruns: 5
     requires:
-        {% for upstream_builder in artifact_completes_builders %}
+        {% for upstream_builder in partner_repacks_upstreams %}
```

Comment:
do we need `partner_repacks_upstreams` or can we just use `artifact_completes_builders` here?